boxes/__init__.py

```diff
@@ -241,7 +243,36 @@ class HexHolesSettings(edges.Settings):
 
     relative_params = {}
 
+class fillHolesSettings(edges.Settings):
+    """Settings for Hole filling
 
+Values:
+
+* absolute
+  * fill_pattern : "hex" : style of hole pattern
+  * hole_style :   "round" : style of hole (does not apply to fill patterns 'vbar' and 'hbar')
+
+* relative (in multiples of thickness)
+
```

Comment:
Yes, using relative feature/hole sizes here is in line with other parts of the libraries and for some use cases having the holes grow with material thickness is just the right thing. But another use case are ventilation holes in front of fans or electronic parts which need to be protected from touch. For those the hole size needs to stay the same. So we either need a way to have both options or move to absolute sizes.

Reply:
so absolute sizes it will be
--> done